libtenzir/builtins/operators/ocsf.cpp

```diff
@@ -890,14 +885,14 @@ class deriver {
         // Both present - just validate consistency
         auto expected_string = enum_lookup.find(*int_value);
         if (expected_string == enum_lookup.end()) {
-          diagnostic::warning("found invalid value for {}", int_name)
+          diagnostic::warning("found invalid value for `{}`", int_path)
             .primary(self_)
             .note("got {}", *int_value)
```

Comment:
```suggestion
            .note("got `{}`", *int_value)
```

According to my search, `got {` appears in a bunch of additional spots as well.

Reply:
I think those are fine. It's not containing some TQL, but an actual value. Same for the strings. Otherwise we would have to write 
```
got `"whatever"` 
```
which looks really weird. But you are right that we are a bit inconsistent and I'm not sure whether it's the best option overall. Will keep this here as-is for now though.